docs/manual/manual_intro.rst

```diff
@@ -10,4 +10,18 @@ Introduction
 This document is meant to provide detailed information on features shared across Vertiq modules. This manual is still being updated, not all features and 
 configurations are covered. For more information and documentation, refer to the `Vertiq website <https://www.vertiq.co/>`_.
 
-This manual is divided into sections based on individual features. Each section specifies which modules support the feature.
\ No newline at end of file
+This manual is divided into sections based on individual features. A listing of modules and their supported features is found below.
+
+.. note:: 
+    
+    Unless otherwise specified, all currently available firmware versions support the listed feature.
+
+.. _supported_features_table:

```

Comment:
This may be a bit hidden in the introduction. You can get here from clicking links, but finding it on its own isn't obvious, and this is the kind of thing people may want to just see. May be worth having its own page here instead of wedging it into the intro, so it's very obvious in the sidebar